### ISSUE TITLE
fig. 3

### ISSUE THREAD

**Issue by tjmahr** [tjmahr/2015_Coartic_Paper#1]
Per reviewer 1, plot figure 3 in empircal logits
